pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -2505,31 +2505,28 @@ class Build {
                 // Run Smoke Tests and AQA Tests
                 if (enableTests) {
                     try {
-                        //Only smoke tests succeed TCK and AQA tests will be triggerred.
-                        if (runSmokeTests() == 'SUCCESS') {
-                            // Remote trigger Eclipse Temurin JCK tests
-                            if (buildConfig.VARIANT == 'temurin' && enableTCK) {
-                                def platform = ''
-                                if (buildConfig.ARCHITECTURE.contains('x64')) {
-                                    platform = 'x86-64_' + buildConfig.TARGET_OS
-                                } else {
-                                    platform = buildConfig.ARCHITECTURE + '_' + buildConfig.TARGET_OS
-                                }           
-                                if ( !(platform  == 'riscv64_linux' || platform =='aarch64_windows') ) {
-                                    if ( !(buildConfig.JAVA_TO_BUILD == 'jdk8u' && platform == 's390x_linux') ) {
-                                        context.echo "Remote trigger Eclipse Temurin AQA_Test_Pipeline job with ${platform} ${buildConfig.JAVA_TO_BUILD}"
-                                        def remoteTargets = remoteTriggerJckTests(platform, filename)
-                                        context.parallel remoteTargets
-                                    }
+                        // Set smoke tests non-blocking
+                        runSmokeTests()
```

Comment:
As per discussion, we should still block on release builds.
```suggestion
                       if (runSmokeTests() == 'SUCCESS' && buildConfig.RELEASE) {
```

Reply:
updated